tests/org.eclipse.n4js.ide.tests/src/org/eclipse/n4js/ide/tests/contentassist/CompletionWithImportsWorkspaceTest.xtend

```diff
@@ -0,0 +1,81 @@
+/**
+ * Copyright (c) 2020 NumberFour AG.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ * Contributors:
+ * NumberFour AG - Initial API and implementation
+ */
+package org.eclipse.n4js.ide.tests.contentassist;
+
+import org.eclipse.n4js.ide.tests.server.AbstractCompletionTest
+import org.junit.Test
+import java.util.List
+
+/**
+ * Code completion tests for scenarios that also might add an import statement
+ * from dependency projects
+ */
+public class CompletionWithImportsWorkspaceTest extends AbstractCompletionTest {
+
+	/** Some default modules that export a number of classes for all tests. */
+	override final List<Pair<String, List<Pair<String, String>>>> getDefaultTestYarnWorkspace() {
+		return #[
+			"P1*" -> #[
+				DEPENDENCIES -> '''
+					«N4JS_RUNTIME_NAME»,
+					P2,
+					SomeNPM,
+					@n4jsd/SomeNPM
+				'''],
+			"P2" -> #[
+				"LibXY" -> '''
+					export public class XY {}
+				'''],
+				
+			NODE_MODULES + N4JS_RUNTIME_NAME -> null,
+			NODE_MODULES + "SomeNPM" -> #[
+				"index"  -> '''//some npm js code'''],
+			NODE_MODULES + "@n4jsd/SomeNPM" -> #[
```

Comment:
Is there already a single place where all options for project creation are documented, i.e. briefly listed (maybe with an example)? Maybe it could be put in a `package-info.java` file in the package where `AbstractIdeTest` is located?

Reply:
(see above)